packages/backend-modules/payments/lib/services/CancellationService.ts

```diff
@@ -63,7 +97,7 @@ export class CancellationService {
           },
         )
 
-    return await this.billingRepo.updateSubscription(
+    const newLocalSub = await this.billingRepo.updateSubscription(
```

Comment:
sorry to be nitpicky, I find the name a bit confusing. Maybe updatedSub or updatedSubscription?

Reply:
I agree, my imagination was a bit limited yesterday 😄